config/routes.rb

```diff
@@ -1,15 +1,15 @@
 Rails.application.routes.draw do
-  get '/booking/', to: 'pages#booking'
-  root 'pages#home'
+  # get '/booking/', to: 'pages#booking'
+  root 'pages#booking'
```

Comment:
[OPTIONAL]

While i say this is optional, but it's sort of something you need to change. The back end server isn't going to be spitting out HTML pages. So we shouldn't be having a "root" route because our routes here should only show our API endpoints (JSON responses).

The complicated part of your project is you didn't separate your Rails API from your React app (should be in separate repos). That's why it's probably going to confuse you later on and this route right here may be an evidence of that.